src/content/11/en/part11d.md

```diff
@@ -125,6 +125,39 @@ For example: We have a project that uses hash based artifact builds for testing,
 
 In the above case, the software we release is tested because the CI system makes sure that tests are run on the code it is about to tag. It would not be incorrect to say that the project uses semantic versioning and simply ignore that the CI system tests individual developer branches/PRs with a hash based naming system. We do this because the version we care about (the one that is released) is given a semantic version.
 
+</div>
+
+<div class="tasks">
+
+### Exercise 11.5
+
+> TODO: make sure exercise number is correct
+
+Let's set up a workflow that will automatically increase (bump) the version when a pull request is merged into master and tag the release with the version number. We will use an open-source action developed by a third-party: `anothrNick/github-tag-action`. You can read the documentation for this action in its [README](https://github.com/anothrNick/github-tag-action).
+
+Start by creating the workflow the same way as in previous exercises. Set push to master as the trigger and use the same environment as in other actions. The workflow should have one job with two steps. The first step is obviously `actions/checkout@v2` as we need have the latest code in the build environment. The second step should be the tagging action itself:
```

Comment:
Maybe we want to avoid saying "obviously"

Reply:
Makes sense, removed that.

Reply:
:+1: